ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+
+### Valuation
+One of the approaches to determining valuation is market references. Experts listed several companies with which they will compare Ubiquity to determine valuation:
+
+- Tea protocol
+
+[https://tea.xyz/](https://tea.xyz/)
+
+- Talent protocol
+
+[https://www.talentprotocol.com/](https://www.talentprotocol.com/)
+
+[https://www.build.top/](https://www.build.top/)
+
+- BuildAI
+
+[https://buildaierc.com/](https://buildaierc.com/)
+
+## Tokenomics
+Tokenomics is one of the most important parts of a project evaluation. From the experts' point of view, the tokenomics of Ubiquity's tokens are opaque and risky. Although the distribution of UBQ tokens is unclear, presumably high inflation of UBQ tokens could lead to a decrease in the value of investments. Investors need to know the main parameters - token distribution, conditions (cliff, vesting, etc), and expected valuation. 
+
+Questions arise regarding the Ubiquity Dollar about mechanisms for maintaining stability, issuance, prevalence, and regulation. Most experts noted that most tokens launched in 2024 are losing their value, and they would like to see a plan for investor exits or for maintaining the value of UBQ.
+
+## Team
+Not counting Sergey, the Ubiquity team is anonymous, which is common for crypto projects. This does not significantly add or reduce investor trust. From the perspective of public promotion, it is necessary to have influencers who will be the voice of the project. Typically, investors want to see a cohesive team because companies with solo entrepreneurs are less stable.
+
+### GitHub
+Most experts highly rated Ubiquity's GitHub. A large number of actively discussed repositories distinctly distinguishes Ubiquity from many projects. One expert suggested comparing the activity with the repositories of popular crypto projects, in particular, Eigen Foundation [https://github.com/eigenfoundation](https://github.com/eigenfoundation).
+
+## Market and Competition
+In its materials, Ubiquity has not clearly shown which market it serves and what problems it solves. Listed problems and solutions do not distinguish Ubiquity from other products. The competitive environment for Ubiquity is incorrectly defined. For each of Ubiquity's product lines, it is necessary to define the competitive environment. Companies described as competitors based on the AI platform criterion are not direct competitors because Ubiquity is not an AI platform. An analysis found no AI products and services offered by Ubiquity. Using LLM via API does not place it alongside companies that train their own models or create solutions using complex prompts. Experts see a wave of projects trying to use the AI + crypto theme. If a project lacks AI competence like Ubiquity, such positioning is perceived negatively.
```

Comment:
I disagree with this point and have not received this as feedback from anybody I pitched. I think that you may not have explained our use cases well. 

Besides, nobody trains their own models anymore, this isn't a valid argument for why an AI company is not valuable (unless you have billions in USD in compute hardware at your disposal.)